docs/topics/native/native-cocoapods.md

```diff
@@ -93,12 +92,61 @@ sudo gem install cocoapods
 
 If you encounter problems during the installation, check the [Possible issues and solutions](#possible-issues-and-solutions) section.
 
-## Add and configure Kotlin CocoaPods Gradle plugin
+## Create a project
+
+If your environment is set up correctly, you can create a new Kotlin Multiplatform project. For that, use the
+Kotlin Multiplatform web wizard or the Kotlin Multiplatform plugin for Android Studio.
+
+### Using web wizard
+
+To create a project using the web wizard and configure the CocoaPods integration:
+
+1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com) and select target platforms for your project.
```

Comment:
I think it's capitalized there because of the headline-style capitalization used across marketing materials. For me, here it's just a common noun, like 'editor'